homeassistant/components/zha/diagnostics.py

```diff
@@ -0,0 +1,80 @@
+"""Provides diagnostics for ZHA."""
+from __future__ import annotations
+
+import dataclasses
+from typing import Any
+
+import bellows
+import pkg_resources
+import zigpy
+from zigpy.config import CONF_NWK_EXTENDED_PAN_ID
+import zigpy_deconz
+import zigpy_xbee
+import zigpy_zigate
+import zigpy_znp
+
+from homeassistant.components.diagnostics.util import async_redact_data
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_UNIQUE_ID
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import device_registry as dr
+
+from .core.const import ATTR_IEEE, DATA_ZHA, DATA_ZHA_CONFIG, DATA_ZHA_GATEWAY
+from .core.device import ZHADevice
+from .core.gateway import ZHAGateway
+from .core.helpers import async_get_zha_device
+
+KEYS_TO_REDACT = {
+    ATTR_IEEE,
+    CONF_UNIQUE_ID,
+    "device_reg_id",
```

Comment:
is this device registry ID in HA? If so, why would that be redacted ?

Reply:
removed the key from the list. Thanks!